reload.go

```diff
@@ -94,19 +98,22 @@ func Do(log func(string, ...interface{}), additional ...dir) error {
 		}
 		if !s.IsDir() {
 			return fmt.Errorf("reload.Do: not a directory: %q; can only watch directories",
-				additional[i].path)
+				a.path)
 		}
 
-		additional[i].path = path
+		a.path = path
```

Comment:
This breaks the watching of "additional paths", since it will no longer be set to the absolute path. I knew I did this somewhat awkward `additional[i]` for a reason, but I missed it in the review (my bad). I'll get it fixed, but just FYI.

Reply:
Sorry, I should've tested more thoroughly.